tests/src/test/scala/com/evolutiongaming/cassandra/sync/CassandraSyncSpec.scala

```diff
@@ -19,25 +21,29 @@ import org.scalatest.wordspec.AnyWordSpec
 
 class CassandraSyncSpec extends AnyWordSpec with BeforeAndAfterAll with Matchers {
   import CassandraSyncSpec._
+  private lazy val cassandraContainer = CassandraContainer(
+    dockerImageNameOverride = DockerImageName.parse("cassandra:3.11.7"),
```

Comment:
is there any particular reason to use this older version?

there is `3.11.17` and also newer ones like `4.1.6` (should be compatible, though I haven't tried), driver-server compatibility details: https://docs.datastax.com/en/driver-matrix/java-drivers.html

Reply:
scassandra uses the 3.*.* version and the migration to 4.* is not that straightforward, so I decided to keep it out of the scope of this PR

Reply:
isn't docker image for the `server`? ``scassandra` is a wrapper around driver and it is OK to keep using driver for version 3.x, but on server we can use newer version